### PR TITLE
[AAP-27887] Initial files for remotes RBAC

### PR DESCRIPTION
**Pull request by nixocio**
Initial files for remotes RBAC

### CODE REVIEW

**Review by vidyanambiar — COMMENTED** [review]
image
 
I successfully applied the galaxy.collection_remote_owner role to user vn2. But the assignments do not show up in the User Access tab.
I do see those assignments returned by the API.

I think the suggested changes below will address this.

File: frontend/hub/administration/remotes/RemotePage/RemoteUserAccess.tsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+  if (error) {
+    return <HubError error={error} handleRefresh={refresh} />;
+  }
+
+  return (
+    <UserAccess
+      service="hub"
+      id={remote?.name?.toString() || ''}
+      type={'remote'}
```
Comment (vidyanambiar) [inline frontend/hub/administration/remotes/RemotePage/RemoteUserAccess.tsx:35]:
```suggestion
      type={'collectionremote'}
```
This is the content-type for remotes and will be used in the content_type__model filter.

File: frontend/hub/administration/remotes/RemotePage/RemoteTeamAccess.tsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+  if (error) {
+    return <HubError error={error} handleRefresh={refresh} />;
+  }
+
+  return (
+    <TeamAccess
+      service="hub"
+      id={remote?.name?.toString() || ''}
+      type={'remote'}
```
Comment (vidyanambiar) [inline frontend/hub/administration/remotes/RemotePage/RemoteTeamAccess.tsx:35]:
```suggestion
      type={'collectionremote'}
```

File: frontend/hub/administration/remotes/RemotePage/RemoteTeamAccess.tsx [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  }
+
+  if (error) {
+    return <HubError error={error} handleRefresh={refresh} />;
+  }
+
+  return (
+    <TeamAccess
+      service="hub"
+      id={remote?.name?.toString() || ''}
```
Comment (vidyanambiar) [inline frontend/hub/administration/remotes/RemotePage/RemoteTeamAccess.tsx:34]: This needs to be the parsed pulp UUID.

File: frontend/hub/administration/remotes/RemotePage/RemoteUserAccess.tsx [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+  }
+
+  if (error) {
+    return <HubError error={error} handleRefresh={refresh} />;
+  }
+
+  return (
+    <UserAccess
+      service="hub"
+      id={remote?.name?.toString() || ''}
```
Comment (vidyanambiar) [inline frontend/hub/administration/remotes/RemotePage/RemoteUserAccess.tsx:34]: This needs to be the parsed pulp UUID.

File: frontend/hub/administration/remotes/RemotePage/RemoteTeamAccess.tsx (hunk 2 above)
Comment (nixocio) [reply frontend/hub/administration/remotes/RemotePage/RemoteTeamAccess.tsx]: The suggested changes, fixed the error above. Ty

**Review by vidyanambiar — APPROVED** [review]
looks great 🎉

**Pull request merged**